internal/services/containers/container_group_resource_test.go

```diff
@@ -1220,7 +1220,7 @@ resource "azurerm_container_group" "test" {
   }
   dns_config {
     nameservers    = ["reddog.microsoft.com", "somecompany.somedomain"]
-    options        = ["one:option", "two:option", "red:option", "blue:option"]
+    options        = null
```

Comment:
Ideally we'll want to have a scenario where we test that whatever we set in the property is actually sent to and returned by the API so I would revert this and leave this test alone 
```suggestion
    options        = ["one:option", "two:option", "red:option", "blue:option"]
```